src/encoded/loadxl.py

```diff
@@ -925,7 +925,8 @@ def store_keys(app, store_access_key, keys, s3_file_name='illnevertell'):
                           SSECustomerAlgorithm='AES256')
 
 
-def load_data(app, access_key_loc=None, indir='inserts', docsdir=None, clear_tables=False):
+def load_data(app, access_key_loc=None, indir='inserts', docsdir=None,
+              clear_tables=False, use_master_inserts=True):
```

Comment:
If use_master_inserts is True, master inserts and indir are combined. That may be worth pointing out in the function comment header.